src/components/QuizGame/index.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+import { fetchQuizQuestions } from "../../components/QuizGameApi";
+import "antd/dist/antd.css";
+import QuestionCard from "../../components/QuizGameQuestions";
+import "./index.css";
+const Difficulty = {
+  EASY: "easy",
+  MEDIUM: "medium",
+  HARD: "hard",
+};
+const TOTAL_QUESTIONS = 10;
+const QuizGame = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [questions, setQuestions] = useState([]);
+  const [currentQuestionNumber, setCurrentQuestionNumber] = useState(0);
+  const [userAnswers, setUserAnswers] = useState([]);
+  const [score, setScore] = useState(0);
+  const [isGameOver, setIsGameOver] = useState(true);
+  const startQuiz = async () => {
+    setIsLoading(true);
+    setIsGameOver(false);
+    const newQuestions = await fetchQuizQuestions(
+      TOTAL_QUESTIONS,
+      Difficulty.EASY
+    );
+    setQuestions(newQuestions);
+    setScore(0);
+    setUserAnswers([]);
+    setCurrentQuestionNumber(0);
+    setIsLoading(false);
+  };
+  const checkAnswer = (e) => {
+    if (!isGameOver) {
+      const answer = e.currentTarget.value;
+      const isCorrect = questions[currentQuestionNumber].correct_answer === answer;
+      if (isCorrect) {
+        setScore((prev) => prev + 1);
+        const answerObject = {
+          question: questions[currentQuestionNumber].question,
```

Comment:
its already in a declared object and the thing is if i take it away to another line with a variable name(which i cant think of something different  than question for naming) it will be more complicated , so we will have in your case variable declared ,object declared and key inside the object declared, i fell it will be harder to understand

Reply:
You can call it `currentQuestion` and it wont get more complicated.

Reply:
or maybe you can change key to `currentQuestion` and repeat same logic for all.